reviews/views.py

```diff
@@ -327,12 +339,20 @@ def get_queryset(self):
         return [value for key, value in decisions.items()]
 
 
-class ReviewDetailView(LoginRequiredMixin, AutoReviewMixin, UserOrSecretaryAllowedMixin,
-                       generic.DetailView):
+class ReviewDetailView(LoginRequiredMixin, AutoReviewMixin,
+                       UsersOrGroupsAllowedMixin, generic.DetailView):
     """
     Shows the Decisions for a Review
     """
     model = Review
+    
+    def get_group_required(self):
+        
+        obj = self.get_object()
+        allowed_groups = [ settings.GROUP_SECRETARY ]
```

Comment:
Kan dit niet beter een oneliner zijn?

``return [ settings.GROUP_SECRETARY, obj.proposal.reviewing_commitee.name ]``

Reply:
Ja, en ik even de variabelenaam consistent gemaakt met de soortgelijke functies. 

Soms vind ik de eerste stijl wel lekker idiomatisch, dat je in íeder geval de secretaris toelaat en daar nog iets aan toevoegt. Maar met maar twee stappen is dat een beetje lelijk.